lib/internal/errors.js

```diff
@@ -69,6 +93,7 @@ module.exports = exports = {
   Error: makeNodeError(Error),
   TypeError: makeNodeError(TypeError),
   RangeError: makeNodeError(RangeError),
+  AssertionError: makeAssertionError(),
```

Comment:
I don't think we're going to need the factory method in this case. The other ones are there just to reduce code duplication. Just define the class and export it directly.